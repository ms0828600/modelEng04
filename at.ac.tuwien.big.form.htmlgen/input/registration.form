<?xml version="1.0" encoding="ISO-8859-1"?>
<form:Form xmi:version="2.0"
    xmlns:xmi="http://www.omg.org/XMI"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:form="http://www.big.tuwien.ac.at/form/1.0"
    welcomePage="//@pages.0">
  <pages title="Registration information">
    <elements
        xsi:type="form:Heading"
        elementId="page0_1"
        content="Registration"
        level="1"/>
    <elements
        xsi:type="form:Heading"
        elementId="page0_2"
        content="Registration conditions"
        level="2"/>
    <elements
        xsi:type="form:Paragraph"
        elementId="page0_3"
        content="To be able to register, you have to fulfill the following conditions:"/>
    <elements
        xsi:type="form:List"
        elementId="page0_4">
      <items
          label="You have to live in Austria."/>
      <items
          label="You have to be current full-time student of a Viennese university."/>
    </elements>
    <elements
        xsi:type="form:Heading"
        elementId="page0_5"
        content="Registration procedure"
        level="2"/>
    <elements
        xsi:type="form:Paragraph"
        elementId="page0_6"
        content="The registration is a two-phase process:"/>
    <elements
        xsi:type="form:List"
        elementId="page0_7"
        ordered="true">
      <items
          label="First you have to fill in the registration form."/>
      <items
          label="Your application will be confirmed by E-Mail."/>
    </elements>
  </pages>
  <pages nextPage="//@pages.2"
      title="Personal data">
    <elements
        xsi:type="form:Heading"
        elementId="page1_h1"
        content="Library registration"
        level="1"/>
    <elements
        xsi:type="form:Heading"
        elementId="page1_h2"
        content="Personal data"
        level="2"/>
    <elements
        xsi:type="form:TextField"
        elementId="page1_q1"
        label="First name"
        mandatory="false"/>
    <elements
        xsi:type="form:TextField"
        elementId="page1_q2"
        label="Last name"
        mandatory="false"/>
    <elements
        xsi:type="form:TextField"
        elementId="page1_q3"
        label="E-mail"/>
    <elements
        xsi:type="form:TextArea"
        elementId="page1_q4"
        label="Address"
        mandatory="false"/>
    <elements
        xsi:type="form:TextField"
        elementId="page1_q5"
        label="Username"/>
    <elements
        xsi:type="form:TextField"
        elementId="page1_q6"
        label="Password"/>
    <elements
        xsi:type="form:SelectionField"
        elementId="page1_q7"
        label="Are you student at a Viennese university?"
        mandatory="false">
      <items
          label="Yes"/>
      <items
          label="No"/>
    </elements>
    <visibilityConditions
        xsi:type="form:SelectionCondition"
        concernsElements="//@pages.2/@elements.2"
        item="//@pages.1/@elements.8/@items.0"/>
    <visibilityConditions
        xsi:type="form:SelectionCondition"
        concernsElements="//@pages.2/@elements.3"
        item="//@pages.1/@elements.8/@items.1"/>
  </pages>
  <pages title="University information"
      previousPage="//@pages.1">
    <elements
        xsi:type="form:Heading"
        elementId="page2_h1"
        content="Library registration"
        level="1"/>
    <elements
        xsi:type="form:Heading"
        elementId="page2_h2"
        content="University information"
        level="2"/>
    <elements
        xsi:type="form:SelectionField"
        elementId="page2_q1_1"
        label="At which university are you studying?"
        mandatory="false"
        selectionFieldType="Checkbox">
      <items
          label="University of Vienna"
          selected="true"/>
      <items
          label="Vienna University of Technology"/>
      <items
          label="Vienna University of Economics and Business"/>
      <items
          label="University of Natural Resources and Life Science"/>
    </elements>
    <elements
        xsi:type="form:TextField"
        elementId="page2_q1_2"
        label="At which university are you studying?"
        mandatory="false"/>
    <elements
        xsi:type="form:SelectionField"
        elementId="page2_q2"
        label="Do you regard the library as important resource for your study?"
        mandatory="false"
        selectionFieldType="Combobox">
      <items
          label="-2"/>
      <items
          label="-1"/>
      <items
          label="0"/>
      <items
          label="1"/>
      <items
          label="2"/>
    </elements>
  </pages>
</form:Form>
