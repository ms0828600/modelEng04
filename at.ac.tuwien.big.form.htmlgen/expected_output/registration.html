<!DOCTYPE html>
<html lang="en">
	<head>
		<title>Registration information</title>
		<meta name="viewport" content="width=device-width, initial-scale=1.0"/>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
		<link href="../assets/form.css" rel="stylesheet" media="screen"/>
		<script src="../assets/jquery-1.10.2.min.js" type="text/javascript"></script>
		<script src="../assets/form.js" type="text/javascript"></script>
		<script type="text/javascript">
		$(document).ready(
			function(){
				form.init();
				form.registerOptional('page1_q1');
				form.registerOptional('page1_q2');
				form.registerMandatory('page1_q3');
				form.registerOptional('page1_q4');
				form.registerMandatory('page1_q5');
				form.registerMandatory('page1_q6');
				form.registerOptional('page1_q7');
				form.registerOptional('page2_q1_1');
				form.registerOptional('page2_q1_2');
				form.registerOptional('page2_q2');
				
				form.registerElement('page0_1');
				form.registerElement('page0_2');
				form.registerElement('page0_3');
				form.registerElement('page0_4');
				form.registerElement('page0_5');
				form.registerElement('page0_6');
				form.registerElement('page0_7');
				form.registerElement('page1_h1');
				form.registerElement('page1_h2');
				form.registerElement('page1_q1');
				form.registerElement('page1_q2');
				form.registerElement('page1_q3');
				form.registerElement('page1_q4');
				form.registerElement('page1_q5');
				form.registerElement('page1_q6');
				form.registerElement('page1_q7');
				form.registerElement('page2_h1');
				form.registerElement('page2_h2');
				form.registerElement('page2_q1_1');
				form.registerElement('page2_q1_2');
				form.registerElement('page2_q2');
				
				form.registerConditional('page1_q7', 'Yes', 
				['page2_q1_1']
				);
				form.registerConditional('page1_q7', 'No', 
				['page2_q1_2']
				);
			});
		</script>
	</head>
	<body>
		<div class="container">
			<div class="page">
				<div class="well">
					<form action="#">
						<fieldset>
							<h1 id="page0_1">Registration</h1>
							<h2 id="page0_2">Registration conditions</h2>
							<p id="page0_3">To be able to register, you have to fulfill the following conditions:</p>
							<ul id="page0_4">
								<li>You have to live in Austria.</li>
								<li>You have to be current full-time student of a Viennese university.</li>
							</ul>
							<h2 id="page0_5">Registration procedure</h2>
							<p id="page0_6">The registration is a two-phase process:</p>
							<ol id="page0_7">
								<li>First you have to fill in the registration form.</li>
								<li>Your application will be confirmed by E-Mail.</li>
							</ol>
						</fieldset>
					</form>
				</div>
			</div>
			<div class="page">
				<div class="well">
					<form action="#">
						<fieldset>
							<h1 id="page1_h1">Library registration</h1>
							<h2 id="page1_h2">Personal data</h2>
							<div class="control-group">
								<label for="page1_q1">First name</label>
								<div class="controls">
									<input type="text" id="page1_q1"/>
								</div>
							</div>
							<div class="control-group">
								<label for="page1_q2">Last name</label>
								<div class="controls">
									<input type="text" id="page1_q2"/>
								</div>
							</div>
							<div class="control-group">
								<label for="page1_q3">E-mail</label>
								<div class="controls">
									<input type="text" id="page1_q3"/>
								</div>
							</div>
							<div class="control-group">
								<label for="page1_q4">Address</label>
								<div class="controls">
									<textarea id="page1_q4"></textarea>
								</div>
							</div>
							<div class="control-group">
								<label for="page1_q5">Username</label>
								<div class="controls">
									<input type="text" id="page1_q5"/>
								</div>
							</div>
							<div class="control-group">
								<label for="page1_q6">Password</label>
								<div class="controls">
									<input type="text" id="page1_q6"/>
								</div>
							</div>
							<div class="control-group">
								<label class="lone" id="label_for_page1_q7">Are you student at a Viennese university?</label>
								<div class="controls" id="page1_q7">
									<label class="radio" for="page1_q7_0">
										<input type="radio" value="Yes" name="page1_q7" id="page1_q7_0" /> Yes
									</label>
									<label class="radio" for="page1_q7_1">
										<input type="radio" value="No" name="page1_q7" id="page1_q7_1" /> No
									</label>
								</div>
							</div>	
						</fieldset>
					</form>
				</div>
			</div>
			<div class="page">
				<div class="well">
					<form action="#">
						<fieldset>
							<h1 id="page2_h1">Library registration</h1>
							<h2 id="page2_h2">University information</h2>
							<div class="control-group">
								<label class="lone" id="label_for_page2_q1_1">At which university are you studying?</label>
								<div class="controls" id="page2_q1_1">
									<label class="checkbox" for="page2_q1_1_0">
										<input type="checkbox" value="University of Vienna" name="page2_q1_1" id="page2_q1_1_0" checked="checked"/> University of Vienna
									</label>
									<label class="checkbox" for="page2_q1_1_1">
										<input type="checkbox" value="Vienna University of Technology" name="page2_q1_1" id="page2_q1_1_1" /> Vienna University of Technology
									</label>
									<label class="checkbox" for="page2_q1_1_2">
										<input type="checkbox" value="Vienna University of Economics and Business" name="page2_q1_1" id="page2_q1_1_2" /> Vienna University of Economics and Business
									</label>
									<label class="checkbox" for="page2_q1_1_3">
										<input type="checkbox" value="University of Natural Resources and Life Science" name="page2_q1_1" id="page2_q1_1_3" /> University of Natural Resources and Life Science
									</label>
								</div>
							</div>
							<div class="control-group">
								<label for="page2_q1_2">At which university are you studying?</label>
								<div class="controls">
									<input type="text" id="page2_q1_2"/>
								</div>
							</div>
							<div class="control-group">
								<label for="page2_q2" class="lone">Do you regard the library as important resource for your study?</label>
								<div class="controls">
									<select id="page2_q2">
										<option value="-2">-2</option>
										<option value="-1">-1</option>
										<option value="0">0</option>
										<option value="1">1</option>
										<option value="2">2</option>
									</select>
								</div>
							</div>	
						</fieldset>
					</form>
				</div>
			</div>
		</div>
	</body>
</html>
